frontend/src/routes/manage/Video/Access.tsx

```diff
@@ -0,0 +1,890 @@
+    const { t } = useTranslation();
+
+    return (
+        <WithTooltip
+            tooltip={t("manage.access.unlisted.explanation")}
+            placement="bottom"
+            tooltipCss={{ width: 400 }}
+            css={{ display: "inline-block" }}
+        >
+            <div css={{
+                fontSize: 14,
+                lineHeight: 1,
+                color: COLORS.neutral60,
+                display: "flex",
+                gap: 4,
+                marginBottom: 16,
+            }}>
+                <FiInfo />
+                {t("manage.access.unlisted.note")}
+            </div>
+        </WithTooltip>
+    );
+};
+
+
+type Action = "read" | "write";
+
+type Option = {
+    value: {
+        roles: string[];
+        action: Action;
+    };
+    label: string;
+}
+
+type AccessUIProps = {
+    event: AuthorizedEvent;
+}
+
+const AccessUI: React.FC<AccessUIProps> = ({ event }) => {
+    const aclSelectRef = useRef<ACLWrapperHandle>(null);
+
+    const initialACL: ACL = {
+        readRoles: event.readRoles as string[],
+        writeRoles: event.writeRoles as string[],
+    };
+
+    return (
+        <div css={{ maxWidth: 1040 }}>
+            <div css={{
+                display: "flex",
+                flexDirection: "column",
+                width: "100%",
+            }}>
+                <ACLSelectWrapper ref={aclSelectRef} {...{ initialACL }} />
+                <ButtonWrapper {...{ aclSelectRef }} />
+            </div>
+        </div>
+    );
+};
+
+type ButtonWrapper = {
+    aclSelectRef: RefObject<ACLWrapperHandle>;
+}
+
+const ButtonWrapper: React.FC<ButtonWrapper> = ({ aclSelectRef }) => {
+    const { t } = useTranslation();
+    const user = useUser();
+    const saveModalRef = useRef<ModalHandle>(null);
+    const resetModalRef = useRef<ModalHandle>(null);
+
+    const containsUser = (acl: ACL) => {
+        const isAdmin = isRealUser(user) && user.roles.includes("ROLE_ADMIN");
+
+        return isAdmin
+            || acl.writeRoles.includes(getUserRole(user))
+            || acl.writeRoles.includes("ROLE_ANONYMOUS")
+            || acl.writeRoles.includes("ROLE_USER");
+    };
+
+    const submit = async (acl: ACL) => {
+        // TODO: Actually save new ACL.
+        // eslint-disable-next-line no-console
+        console.log(acl);
+    };
+
+    return <div css={{ display: "flex", gap: 8, alignSelf: "flex-start", marginTop: 40 }}>
+        {/* Reset button */}
+        <ButtonWithModal
+            buttonKind="danger"
+            modalRef={resetModalRef}
+            label={t("manage.access.reset-modal.label")}
+            title={t("manage.access.reset-modal.title")}
+            body={t("manage.access.reset-modal.body")}
+            confirmationLabel={t("manage.access.reset-modal.label")}
+            handleClick={() => currentRef(resetModalRef).open()}
+            onConfirm={() => aclSelectRef.current?.reset?.()}
+        />
+        {/* Save button */}
+        <ButtonWithModal
```

Comment:
This one should be greyed out as long as nothing has changed. (Ideally, it is also greyed out if a user changes something, then undoes that and arrives at the initial ACL again; but that's just a nice to have)